cmd/agent/subcommands/integrations/command.go

```diff
@@ -129,16 +174,25 @@ You must specify a version of the package to install using the syntax: <package>
 		Args:  cobra.ExactArgs(1),
 		Long:  ``,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return show(globalParams, cmd, args)
+			cliParams.args = args
+			return fxutil.OneShot(show,
+				fx.Supply(cliParams),
+				fx.Supply(core.BundleParams{
+					ConfFilePath:      globalParams.ConfFilePath,
+					ConfigLoadSecrets: false,
```

Comment:
Question: When using `core.BundleParams` how to know the correctly value for `ConfigLoadSecrets` and `ConfigMissingOK`?